docs/docs/product-documentation.md

```diff
@@ -75,54 +74,58 @@ The strategic aim of the Swapchain team is to develop an HTLC-backed cryptocurre
 
 Intuitivity:  
-Second, the Swapchain application is expected to provide an intuitive, easy-to-use interface. While related to the first aspect (simplicity), ease-of-use also includes the necessity of aspects such as obvious calls to action, for example if users needs to register for an account first or want to quickly browse existing swap offers. Moreover, users must be enabled to give input into the application without circumstances, which could for instance appear in the form of autofill patterns if they repeatedly want to set up the same swap offer. Moreover, barrier-free accessibility is also an important topic to be covered, however this functionality makes sense to be rolled out only when all other basic features have been implemented. Then, voice commands (of course excluding sensitive parts such as private keys) could prove viable support for impaired people.
+Second, the Swapchain application is expected to provide an intuitive, easy-to-use interface. While related to the first aspect (simplicity), ease-of-use also includes the necessity of aspects such as obvious call to action to start trading. Moreover, users must be enabled to give input into the application without circumstances, which could for instance appear in the form of autofill patterns if they repeatedly want to set up the same swap offer. Moreover, barrier-free accessibility is also an important topic to be covered, however this functionality makes sense to be rolled out only when all other basic features have been implemented. Voice commands (of course excluding sensitive parts such as private keys) could prove viable support for impaired people imaginably. 
 
-Adaptability:  
+Adaptability: 
+//den Teil verstehe ich nicht so recht, lass uns da noch mal drüber quatschen 
 With more sophisticated versions of the software, users are given more opportunities to adapt the software design according to their own needs. This might include an activatable retrospective and analysis on prior transactions if the user wishes to receive in-depth intel on previous swaps.
 
 Reliability:  
-The user interface must clearly inform the user about any important event regarding the pending transactions. This includes understandable notifications, for example when a refund is granted due to an exceeded timelock or if a transaction was successful. Furthermore, users must be given obstacle-free opportunities to receive timely support, especially in an area as delicate as assets trading.
+The user interface must clearly inform the user about any important event regarding the pending transactions. This includes understandable notifications, for example when a refund is granted due to an exceeded timelock, if a transaction was successful or whether the process is still ongoing and therefore the user needs to stay in the browser. Furthermore, users must be given obstacle-free opportunities to receive timely support, especially in an area as delicate as assets trading.
```

Comment:
Would also be good to include another point "Flexibility", as the users can choose their timelocks and also the fees they want and also the cryptocurrencies they would like to swap.